packages/hop-node/src/config/fileOps.ts

```diff
@@ -173,11 +203,37 @@ export async function setGlobalConfigFromConfigFile (
   if (config?.metrics) {
     setMetricsConfig(config.metrics)
   }
-  if (config?.fees) {
+  if (!config?.routes) {
+    throw new Error('config for routes is required')
+  }
+  const numRoutes = Object.keys(config.routes).length
+  if (!numRoutes) {
+    throw new Error('1 or more routes must be specified')
+  }
+  if (!config?.watchers) {
+    throw new Error('config for watchers is required')
+  }
+
+  const enabledWatchers = Object.keys(config?.watchers).filter((watcher: string) => (config?.watchers as any)?.[watcher])
+  if (!config?.watchers) {
+    throw new Error('config for watchers is required')
+  }
+
+  setRoutesConfig(config.routes)
+  if (enabledWatchers.includes(Watchers.BondWithdrawal) && !config?.fees) {
+    throw new Error('config for fees is required')
+  }
+
+  if (config.fees != null) {
```

Comment:
Why does this `if` statement do nothing if it does not exist but the others throw (i.e., why does `if (!config.watchers)` throw but this does not)?

Reply:
It's because `fees` is only required if the `bondWithdrawal` watcher is enabled, while others, like `watchers`, is always required. The validation checks the values of `fees`